[docs] Faucet update (#20958)
MystenLabs/sui: docs/content/snippets/faucet-online.mdx

> Get test SUI from the online faucet at https://faucet.sui.io/. You can refresh your browser to perform multiple requests, but the requests are rate-limited per IP address.

Reviewer: ```suggestion
Visit the online faucet to request SUI tokens: https://faucet.sui.io/. You can refresh your browser to perform multiple requests, but the requests are rate-limited per IP address.
```
